### ISSUE TITLE
Build process review

### ISSUE THREAD

**Issue by lamara11** [amosproj/amos2024ss01-xcelerator-demo-app#31]
## User story

1. As a developer
2. I need to be ready to demo my project’s build process

## Acceptance criteria

- [x] The demo is maximum 5 minutes
- [x] The demo states what the project is about
- [x] The demo states what modules there are, if more than one
- [x] The demo states what build tool technology the developer is using
- [x] The demo states what build targets / tasks there are

## Definition of done (DoD)

- [x] Added only after week 5
- [x] The same for all features
- [x] Here goes the project specific part

## DoD general criteria

- [x] Feature has been fully implemented
- [x] Feature has been merged into the mainline
- [x] All acceptance criteria were met
- [x] Product owner approved features
- [x] All tests are passing
- [x] Developers agreed to release
